google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/TableAdminExample.java

```diff
@@ -0,0 +1,373 @@
+    }
+    // [END bigtable_get_table_metadata]
+  }
+
+  /** Demonstrates how to create a new instance of the DurationRule. */
+  public void maxAgeRule() {
+    System.out.printf("%nCreating column family %s with max age GC rule%n", COLUMN_FAMILY_1);
+    // [START bigtable_create_family_gc_max_age]
+    // Creates a column family with GC policy : maximum age
+    // where age = current time minus cell timestamp
+
+    // Defines the GC rule to retain data with max age of 5 days.
+    GCRules.DurationRule maxAgeRule1 = GCRules.GCRULES.maxAge(5, TimeUnit.DAYS);
+
+    // Creates column family with given GC rule.
+    try {
+      ModifyColumnFamiliesRequest columnFamiliesRequest1 =
+          ModifyColumnFamiliesRequest.of(tableId).addFamily(COLUMN_FAMILY_1, maxAgeRule1);
+      adminClient.modifyFamilies(columnFamiliesRequest1);
+      System.out.println("Created column family: " + COLUMN_FAMILY_1);
+    } catch (AlreadyExistsException e) {
+      System.err.println(
+          "Failed to create column family with rule, already exists: " + e.getMessage());
+    }
+    // [END bigtable_create_family_gc_max_age]
+  }
+
+  /** Demonstrates how to create a new instance of the VersionRule. */
+  public void maxVersionsRule() {
+    System.out.printf("%nCreating column family %s with max versions GC rule%n", COLUMN_FAMILY_2);
+    // [START bigtable_create_family_gc_max_versions]
+    // Creates a column family with GC policy : most recent N versions
+    // where 1 = most recent version
+
+    // Defines the GC policy to retain only the most recent 2 versions.
+    GCRules.VersionRule versionRule1 = GCRules.GCRULES.maxVersions(2);
+
+    // Creates column family with given GC rule.
+    try {
+      ModifyColumnFamiliesRequest columnFamiliesRequest2 =
+          ModifyColumnFamiliesRequest.of(tableId).addFamily(COLUMN_FAMILY_2, versionRule1);
+      adminClient.modifyFamilies(columnFamiliesRequest2);
+      System.out.println("Created column family: " + COLUMN_FAMILY_2);
+    } catch (AlreadyExistsException e) {
+      System.err.println(
+          "Failed to create column family with rule, already exists: " + e.getMessage());
+    }
+    // [END bigtable_create_family_gc_max_versions]
+  }
+
+  /** Demonstrates how to create a new instance of the UnionRule. */
+  public void unionRule() {
+    System.out.printf("%nCreating column family %s with union GC rule%n", COLUMN_FAMILY_3);
+    // [START bigtable_create_family_gc_union]
+    // Creates a column family with GC policy to drop data that matches at least one condition.
+
+    // Defines a GC rule to drop cells older than 5 days OR not the most recent version.
+    GCRules.DurationRule maxAgeRule2 = GCRules.GCRULES.maxAge(5, TimeUnit.DAYS);
+    GCRules.VersionRule versionRule2 = GCRules.GCRULES.maxVersions(1);
+    // Add rules to union rule list
+    GCRules.UnionRule unionRule1 = GCRules.GCRULES.union().rule(maxAgeRule2).rule(versionRule2);
```

Comment:
Please import UnionRule directly

Also please inline the creation of the child rules:

```
UnionRule unionRUle = GCRULES.union()
  .rule(GCRULES.maxAge(5, TimeUnit.DAYS))
  .rule(GCRULES.maxVersions(1))
```